apps/console/src/features/users/components/bulk-import-response-list.tsx

```diff
@@ -163,10 +212,14 @@ export const BulkImportResponseList: React.FunctionComponent<BulkImportResponseL
      * Filters the response list based on the search query and the selected status.
      */
     const filterBulkUsersResponseList = () => {
-        let filteredList: BulkUserImportOperationResponse[] = responseList;
+        let filteredList: BulkUserImportOperationResponse[] = responseList.filter(
```

Comment:
Let's check and add nullable checks wherever necessary

Reply:
Added